idioms-json-2.1-extensions/incident-malware.json

```diff
@@ -22,14 +22,35 @@
             "extensions": {
                 "extension-definition--7a8eaf47-9b0f-487d-b280-1e6cc4cccee9": {
                     "contacts": [
-                        "identity--8e5febda-ffd0-4ade-8afe-9a7e64894510",
-                        "identity--b2557302-99e3-496a-825f-8e8c5501bec8"
+                        "identity--6e85a81b-5fd8-46c0-ba46-df128771f695",
+                        "identity--c0cdd136-daa0-40ef-8957-a0e686fcd0d2"
+                    ],
+                    "discovery_methods": [
+                        "Log Review",
+                        "Antivirus"
                     ],
                     "extension_type": "property-extension",
+                    "intended_effects": [
+                        {
+                            "Degradation of Service": {
+                                "description": "Bring down server"
+                            }
+                        }
+                    ],
```

Comment:
Something looks weird about this one.... deeply nested structure. Would in it be:

``` json
{
    "value": "Degradation of Service",
    "description": "Bring down server"
}
```

Reply:
I took care of this